api/rally2-matching-system/matching-system-swagger.yml

```diff
@@ -58,34 +51,25 @@ definitions:
   Template:
     type: object
     description: |
-      Object that contains a byte array of template data.
+      Data transfer object for a template.
     properties:
       Template:
-        description: A byte array that contains template data.
-        type: array
+        description: A base64 encoded string that contains template data.
+        type: string
+        format: byte
+        example: dGhpcyBzZW50ZW5jZSBpcyBhbiBleGFtcGxlIHRlbXBsYXRlLi4K
         x-nullable: false
-        items:
-          type: string
-          format: byte
 
   Comparison:
     type: object
     description: |
-      Comparison contains a similarity score for a single templare comparison operation.
+      A similarity score for a single (1:1) template comparison operation.
     properties:
-      AlgorithmName:
-        type: string
-        description: Name of the algorithm used to create this comparison
-        x-nullable: false
-      NormalizedScore:
-        type: number
-        format: float
-        description: Similarity score between 0 and 1, with 1 being the highest score the algorithm can produce
-        x-nullable: false
       Score:
         type: number
         format: float
-        description: Un-normalized similarity score, raw score as produced by the algorithm
+        description: An un-normalized similarity score, as produced by the algorithm
```

Comment:
Where did the normalized score go? I feel like letting them go unbounded and only unbounded could cause issues

Reply:
Do we want both?
I thought we specifically didn't want altered/weighted scores.

If it really is a pure 0-1 normalization, then it would be fine... but it does invite the option to weight the scores in other ways. 

I can add it back in if you'd like.
Up to you.

Reply:
sounds good.